test/connectivity/README.md

```diff
@@ -69,3 +72,106 @@ Network offline/online should not affect this test result since Edge hub stores
 6. When DeploymentTester2 starts up, it will get all environment variables and report to Test Result Coordinator (#5).
 
 When network goes offline, route#3 will not happen.  But once the network comes back online, Edge Agent should be able to get the latest updated deployment and deploy DeploymentTester2.
+
+#### Cloud to Module Direct Method Test
+
+![Connectivity Test Diagram](./images/ConnectivityTest_C2M_DirectMethod.png)
+
+The Direct Method Connectivity Test is designed to test direct method resilience against network connectivity issue. 
+##### Test Specific Component:
+	1. Direct Method (Cloud) Sender is an Azure service client that is wrapped in a container module that is deployed into a test agent.  It is responsible for sending a "HelloWorldMethod" direct method to the Direct Method Receiver module and report the direct method status to Test Result Coordinator
```

Comment:
add period at the end.

Reply:
DM Cloud sender is in host network, do you think we need to explicit mention it here.